--- gcc-linaro-git-131f0279/libcc1/configure.orig	2018-10-18 15:38:07.975033000 +0200
+++ gcc-linaro-git-131f0279/libcc1/configure	2018-10-18 15:38:43.151098700 +0200
@@ -14580,7 +14580,7 @@
        if test x$build = x$host; then
 	 export_sym_check="objdump${exeext} -T"
        elif test x$host = x$target; then
-	 export_sym_check="$gcc_cv_objdump -T"
+	 export_sym_check="${target}-objdump -T"
        else
 	 export_sym_check=
        fi
